spec/system/page_objects/modals/dismiss_new.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Modals
+    class DismissNew < PageObjects::Modals::Base
+      def has_dismiss_topics_checked?
+        find(".dismiss-topics input")["checked"] == "true"
```

Comment:
This pattern has a chance of being flaky because `find` will only wait for the element with the specified class to appear but not wait until it is "checked".